src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,5 +1,7 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
```

Comment:
This code section is a Javadoc of the login() method and should be right above it, class fields and constructors on the other hand should be at the beginning of the class